src/public/app/widgets/note_title.js

```diff
@@ -38,7 +38,9 @@ export default class NoteTitleWidget extends NoteContextAwareWidget {
 
             protectedSessionHolder.touchProtectedSessionIfNecessary(this.note);
 
-            await server.put(`notes/${this.noteId}/title`, {title}, this.componentId);
+            // Regex is making sure the title is not full of spaces
+            if ( title.match("^(?!\\s*$)"))
```

Comment:
I think we can replace the regex with something a little bit more efficient:

```suggestion
            if (title.trim())
```

Also please use curly braces for `if`s, even if they only have one statement.